app/src/main/java/gov/hhs/cdc/trustedintermediary/external/javalin/DomainsRegistration.java

```diff
@@ -15,9 +16,17 @@
 import java.util.stream.Collectors;
 
 public class DomainsRegistration {
+
+    private static Slf4jLogger logger = ApplicationContext.getImplementation(Slf4jLogger.class);
+
     public static void registerDomains(
             Javalin app, Set<Class<? extends DomainConnector>> domainConnectors) {
 
+        logger.logInfo("Info");
+        logger.logWarning("Warning");
+        logger.logDebug("Debug");
+        logger.logError("Error");
```

Comment:
Were these here for testing?

Reply:
Yeah, those come up when the app is fired up. I can take them down, it's only a reference for the color.